.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,74 @@
+name: Bug report
+description: Create a report describing unexpected or incorrect behavior in astropy.
+labels: Bug
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Thanks for taking the time to fill out this bug report!
+        Please be sure to check out our
+        [contributing guidelines](https://github.com/astropy/astropy/blob/main/CONTRIBUTING.md)
+        and [code of conduct](https://github.com/astropy/astropy/blob/main/CODE_OF_CONDUCT.md).
+        Please have a search on our GitHub repository to see if a similar
+        issue has already been posted. If a similar issue is closed, have a
+        quick look to see if you are satisfied by the resolution.
+        If not please go ahead and open an issue!
+        Please check that the development version still produces the same bug.
+        You can install development version with
+        `pip install git+https://github.com/astropy/astropy` command.
```

Comment:
Is it a good idea to do this outside a virtual environment? Maybe it would be better to provide a link to https://docs.astropy.org/en/latest/development/workflow/get_devel_version.html instead?